modules/labs-react/toast/stories/examples/Basic.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import {Toast} from '@workday/canvas-kit-labs-react/toast';
+import {checkIcon} from '@workday/canvas-system-icons-web';
+import {colors} from '@workday/canvas-kit-react/tokens';
+
+export const Basic = () => (
+  <Toast>
+    <Toast.Icon icon={checkIcon} color={colors.greenApple400} />
```

Comment:
Comment (not an RC): It would be nice if this prop was a token-aware, so I could do this:

```tsx
<Toast.Icon icon={checkIcon} color="greenApple400" />
```

But we'd need to make some adjustments to `SystemIcon` for that to work.

- Adjust the color type in `SystemIconStyles`
  - `string` becomes `CanvasColor | (string & {});`
- Remove `color` from the Omit props
- Add a function to process the colors

```ts
import {iconColors, colors, CanvasColor} from '@workday/canvas-kit-react/tokens';

function getColor(value: CanvasColor | string) {
  if (value in colors) {
    return colors[value as keyof typeof colors];
  }
  return value;
}
```

Reply:
https://github.com/Workday/canvas-kit/issues/1751